go.mod

```diff
@@ -5,12 +5,12 @@ go 1.18
 require (
 	github.com/golang/protobuf v1.5.3
 	google.golang.org/grpc v1.57.0
+	google.golang.org/protobuf v1.31.0
```

Comment:
module github.com/golang/protobuf is deprecated.  Since you added "google.golang.org/protobuf", can you remove "github.com/golang/protobuf"?

Reply:
removed